src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Packaging/ManagedProjectSystemClientPackage.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.Packaging
+{
+    [Guid(PackageGuid)]
+    [PackageRegistration(AllowsBackgroundLoading = true, RegisterUsing = RegistrationMethod.CodeBase, UseManagedResourcesOnly = true)]
+
+    internal sealed class ManagedProjectSystemClientPackage : AsyncPackage
```

Comment:
Why do we need a package? I think we should only add it if actuallay needed.

Reply:
We don't need a package to produce a pkgdef, if that's why you added it.

Reply:
I'm going to need it later when I start defining menu items and need a place to put `[ProvideMenuResource("Menus.ctmenu", 1)]`. At least, I think I do.

Reply:
Can we wait until then before we check this in? I just deleted a bunch of pre-emptive packages.